twine/commands/upload.py

```diff
@@ -29,6 +29,7 @@
 import pkginfo
 import pkg_resources
 import requests
+from requests_toolbelt import MultipartEncoder
```

Comment:
Importing things from requests_toolbelt is considered deprecated. This should import from `requests_toolbelt.multipart`.